camelot/types/__init__.py

```diff
@@ -341,6 +342,29 @@ def python_type(self):
     def __repr__(self):
         return 'File()'
 
+
+class HashFile(StoredFile):
```

Comment:
?? er moet mi geen subclass van StoredFile komen, maar een subclass van 'File'

Reply:
Ik heb nergens gevonden in de code en online of je een custom type kunt maken waarbij je andere kolommen accessed, dus daarom geen subclass van file.